assets/javascripts/tests.js

```diff
@@ -264,7 +264,11 @@ function renderTestLists() {
       data: ajaxQueryParams,
       dataSrc: function (json) {
         // update heading when JSON is available
-        $('#running_jobs_heading').text(json.data.length + ' jobs are running');
+        let text = json.data.length + ' jobs are running';
+        if (json.max_running_jobs >= 0) {
+          text += ' (limited by server config)';
```

Comment:
How about `x/y jobs are running` where **x** is the number of jobs and **y** is the configured maximum

Reply:
https://github.com/os-autoinst/openQA/pull/5279#issuecomment-1673322996